server/migrations/20190107234122_initial.js

```diff
@@ -0,0 +1,55 @@
+exports.up = function(knex, Promise) {
+  return Promise.all([
+    knex.schema.createTable('companies', (table) => {
+      table.increments('id').primary();
+      table.text('name').notNullable();
+      table.text('description').notNullable();
+      table.specificType('industries', 'text ARRAY');
+      table.text('status').notNullable();
+      table.text('contact_email').notNullable();
+      table.json('company_links').defaultTo('{}');
+      table.timestamps();
+    }),
+    knex.schema.createTable('users', (table) => {
```

Comment:
Missing role key

Reply:
I renamed it to `permissions` per default for [feathers-permissions](
https://github.com/feathersjs-ecosystem/feathers-permissions), which encompasses roles as well.